<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yakuza 0 Save Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="kiryu_tab">
     <attribute name="title">
      <string>Kiryu</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Money:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Styles</string>
      </property>
      <widget class="QCheckBox" name="style_dod">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>DoD</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="style_beast">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Beast</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="style_rush">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Rush</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="kiryu_money">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Outfit</string>
      </property>
      <widget class="QCheckBox" name="outfit_dod">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>DoD</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="outfit_dragon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Dragon</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="outfit_judgement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Judgement</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="outfit_producer">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Producer</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="outfit_new_hire">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>New Hire</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="majima_tab">
     <attribute name="title">
      <string>Majima</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Money:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>151</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Styles</string>
      </property>
      <widget class="QCheckBox" name="style_mdos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>MDoS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="style_breaker">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Breaker</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="style_slugger">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>86</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Slugger</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="majima_money">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>General</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Character:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Difficulty</string>
     </property>
    </widget>
    <widget class="QComboBox" name="difficulty">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Easy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extreme</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="current_char">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Kiryu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Majima</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>load_button</zorder>
   <zorder>tabWidget</zorder>
   <zorder>save_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
